### PR TITLE
Make short URLs case insensitive.

### LINKED ISSUE #1: Make short URLs case insentive
In most cases the reason for using this service is that you don't want computer generated hashes as short urls. Therefore, you probably also don't want to have case sensitive short urls.

`/foo` should resolve to the same thing as `/FOO` AND `/Foo`

### PR DESCRIPTION
**Pull request by andylolz**
I’m not exactly sure this handles the analytics the way you might want it to…

Fixes #1.

### CODE REVIEW

File: plinky.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def redirect_to_short_url(shorturl=None):
     'shorturl': shorturl
   })
 
-  return redirect(shortcuts.lookup_shorturl(shorturl), 302)
+  return redirect(shortcuts.lookup_shorturl(shorturl.lower()), 302)
```
Comment (martinpeck) [inline plinky.py]: This should be hidden within shortcuts.py, so that all of the case-insensitive stuff is consolidated in that class.

Comment (andylolz) [inline plinky.py]: Good point, yes

**Comment by martinpeck** [comment]
Thanks. I'm going to have a think about how I want the analytics to work before I merge this. I suspect it might be better to return both the short (and 'canonical') url, plus the long url, as a tuple from the lookup and then log both values (what was asked for, and what was given)

**Comment by martinpeck** [comment]
Meh. Analytics can wait for another day. Thanks @andylolz

**Pull request merged**